### PR TITLE
New Run3 offline GT

### PR DESCRIPTION
**Pull request by francescobrivio**
#### PR description:
This PR introduces a new key in autoCond to store the offline reprocessing GT for Run3, announced in the [AlCaDB HN](https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/4459.html).
The new GT is: [113X_dataRun3_v1](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/113X_dataRun3_v1)
The difference wrt dataRun2 GT is: https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/113X_dataRun2_v6/113X_dataRun3_v1

The PR also introduces a new relval workflow `136.897` that consumes this new GT.
I have some questions about this new workflow for @cms-sw/pdmv-l2 : 
- Is the number of the new workflow ok?
- Should we add this to the standard matrix?

Finally, I modified the names of the steps of the Prompt relval workflow (`138.1`) since they consume the prompt GT and they shoul not be confused with the steps of the newly introduced Run3 offline workflow.

**[EDIT]**
As described in [JIRA/PDMVRERECO-46](https://its.cern.ch/jira/browse/PDMVRERECO-46?focusedCommentId=3967472&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-3967472) the new GT has already been update to v2 to include two new payloads for `SiStripApvGain_FromParticles_GR10_v12_offline` and `TrackerSurfaceDeformations_v13_offline`.
The new GT is: [113X_dataRun3_v2](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/113X_dataRun3_v2)
And the difference wrt to v1 is: [113X_dataRun3_v2 vs 113X_dataRun3_v1](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/113X_dataRun3_v2/113X_dataRun3_v1)

#### PR validation:
Tested with: `runTheMatrix.py -l 138.1,136.897 --ibeos -j4`

#### Backport:
Once this PR is signed off we will make backports to 12_0_X and 11_3_X

### CODE REVIEW

**Comment by francescobrivio** [comment]
Test parameters:
  - worfklows=138.1,136.897

**Comment by francescobrivio** [comment]
Test parameters:
   - workflows=138.1,136.897

**Comment by AdrianoDee** [comment]
test parameters:
- worflows=138.1,136.897

**Comment by francescobrivio** [comment]
@cmsbuild please test

**Comment by AdrianoDee** [comment]
please test

**Comment by francescobrivio** [comment]
> test parameters:
> 
>     * worflows=138.1,136.897

@AdrianoDee I'm not sure this got processed correctly by the bot due to the typo...

**Comment by AdrianoDee** [comment]
please abort

**Comment by tvami** [comment]
> Test parameters:
> 
> * workflows=138.1,136.897

this would have worked

**Comment by tvami** [comment]
test parameters:
  - workflows = 138.1,136.897

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by AdrianoDee** [comment]
Sorry, we perfectly crossed and I'd missed your commands.

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -493,6 +493,9 @@
 ### LS2 - MWGR 2021 - CSC, DAQ, DCS, DQM, DT, ECAL, GEM, HCAL, L1SCOUT, PIXEL, RPC, TCDS, TRACKER, TRG ###
 steps['RunCosmics2021']={'INPUT':InputInfo(dataSet='/ExpressCosmics/Commissioning2021-Express-v1/FEVT',label='2021GR5',ls={343498: [[1, 10000]]},events=100000,location='STD')}
 
+#### run3 Cosmics ####
+##Run 344518 @ 0T 2021CRUZET
+steps['RunCosmics2021CRUZET']={'INPUT':InputInfo(dataSet='/Cosmics/Commissioning2021-v1/RAW',label='2021CRUZET',events=100000,location='STD')}
```
Comment (tvami) [inline Configuration/PyReleaseValidation/python/relval_steps.py:498]: @francescobrivio doesn't this miss the `,ls={344518: [[1, 10000]]},` part?

Comment (francescobrivio) [reply Configuration/PyReleaseValidation/python/relval_steps.py:498]: I copied it from the Run2 cosmics case: https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_steps.py#L487-L491 which don't have limitations on the number of ls, only the MWGR has it...not sure why!

Comment (tvami) [reply Configuration/PyReleaseValidation/python/relval_steps.py:498]: but wait the Run-2 line has `,run=[256259]`, I dont see the equivalent of that in here

Comment (francescobrivio) [reply Configuration/PyReleaseValidation/python/relval_steps.py:498]: ah good catch! I must have deleted it by mistake when I removed the ls range I was using for test. Fix coming right away...

**Comment by tvami** [comment]
@cmsbuild , please test

**Comment by francescobrivio** [comment]
@perrotta @qliphy @smuzaffar the DAS error is for the newly introduced workflow which apparently can not find the needed files although we made an apposite Rucio rule to make sure that the files are available on `T2_CH_CERN`.
And indeed if I run:
```
dasgoclient --query 'file dataset=/Cosmics/Commissioning2021-v1/RAW run=344518 site=T2_CH_CERN'
```
I find 158 files.

From the [cmdLog](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-909f00/18430/runTheMatrixINPUT-results/136.897_RunCosmics2021CRUZET+RunCosmics2021CRUZET+RECOCOSDRUN3+ALCACOSDRUN3+HARVESTDCRUN3/cmdLog) I can see the tests are looking for the correct files, but the [step1_dasquery.log](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-909f00/18430/runTheMatrixINPUT-results/136.897_RunCosmics2021CRUZET+RunCosmics2021CRUZET+RECOCOSDRUN3+ALCACOSDRUN3+HARVESTDCRUN3/step1_dasquery.log) is empty...
Should the files be added somewhere else to be picked up for the matrix tests?

**Comment by mmusich** [comment]
@francescobrivio 

> This PR introduces a new key in autoCond to store the offline reprocessing GT for Run

I guess a 3 is missing in the PR description.
Also can you please post in it a diff of the Run2 / Run3 offline data keys?
Thx

**Comment by qliphy** [comment]
@francescobrivio   
@smuzaffar may comment more, 
This might be related to:
https://github.com/cms-sw/cmssw/pull/33090#issuecomment-792808610
Probably you just need to try to test again in a few hours.

**Comment by smuzaffar** [comment]
no need to re-runthe full tests, I just have restarted the relval-input test  and hopefully this time it should find the files for this dataset ( our caching system has already cached it https://github.com/cms-sw/cms-sw.github.io/blob/master/das_queries/9c/9c2362bd54d022117eb56110e4ff0ca92678c902f2bd6e03de86b973d43108e5 )

**Comment by tvami** [comment]
I don't see wf 136.897 in the bin-by-bin comparison:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_12_1_X_2021-09-08-1100+909f00/45298/dqm-histo-comparison-summary.html

Is that because this is a new wf so there is no baseline? It would be still nice to see plots for the PR alone...

**Comment by smuzaffar** [comment]
@tvami , relval comparison is only done for very selected number of existsing workflows (`runTheMatrix.py -s` + few more)

**Comment by tvami** [comment]
+alca

* the GT doesn't break the code, without the bin-by-bin comparison there is not much to say
* we also did a diff wrt the prompt GT and everything looks fine:
https://docs.google.com/spreadsheets/d/1s4bnsQwQzNscjcCehIk4sLRbOJorbjT3Ejp7nUZp5LI/edit#gid=1041337917

**Comment by tvami** [comment]
@cms-sw/pdmv-l2 are you guys ok with the wf naming?

**Comment by francescobrivio** [comment]
Test parameters:
  - workflow = 136.897

(the last commit does not change the promtp GT, thus no need to re-test `138.1`)

**Comment by francescobrivio** [comment]
Test parameters:
  -  workflow = 136.897

**Comment by francescobrivio** [comment]
@cmsbuild please test

**Comment by tvami** [comment]
+alca

**Comment by srimanob** [comment]
+Upgrade

**Comment by kskovpen** [comment]
+1

**Comment by perrotta** [comment]
> @cms-sw/pdmv-l2 are you guys ok with the wf naming?

I take the "+1" from PDMV as a "Yes"...

**Comment by perrotta** [comment]
+1

**Comment by kskovpen** [comment]
> > @cms-sw/pdmv-l2 are you guys ok with the wf naming?
> 
> I take the "+1" from PDMV as a "Yes"...

Yes, sorry, was in a robot mode. Unless @bbilin and @jordan-martins think otherwise.

**Pull request merged**